virt/monitoring/virt-runbooks.adoc

```diff
@@ -6,119 +6,296 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-
 :!virt-runbooks:
-You can use the procedures in these runbooks to diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts].
+Runbooks for the {VirtProductName} Operator are maintained in the link:https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[openshift/runbooks] Git repository and can be viewed on GitHub. You can use the procedures in these runbooks to diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts].
```

Comment:
Leaving the first part as-is because "The Git repository maintains the runbooks" sounds like the git repository itself is maintaining the runbooks (if only!). 😹

I might take the second part of your suggestion and try to work in "You can view..." 
Thanks!